.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
 venv
+service/__pycache__
```

Comment:
Not a change worthy issue, but typically I start projects by googling "{{language}} gitignore" so "python gitignore". You'll quickly find [this](https://github.com/github/gitignore/blob/master/Python.gitignore). Little tricks in there that will catch pretty much everything you do/will want. For example, its already setup to automatically exclude all `__pycache__` dirs anywhere in the project.